### DISCUSSION TITLE
--enable-unsafe-webgpu ?

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#685] [Q&A] [answered]
Since the addition of `--enable-unsafe-webgpu` to `chrome-flags.conf` chromium-based browsers display a warning message below the bookmarks bar:

![swappy-20240124_130444](https://github.com/manjaro-sway/manjaro-sway/assets/274236/ee1deb23-638c-416a-af87-826cdd9cab36)

Is that intended behavior? What is the benefit of this setting if it is unsupported and allegedly unstable?

**Comment by boredland** [comment, marked as answer]
We removed it. Some chromium things need it.
